7_react/react1/week2/todo-app/src/todoList/TodoList.jsx

```diff
@@ -0,0 +1,53 @@
+import TimeCount from "../timer/Timer";
+import TodoItem from "../todoItem/TodoItem";
+import todos from "../todoItem/todos";
+import { useState } from "react";
+
+function AppTodo() {
+  // const data = [...todos];
+  const [todolist, setTodoList] = useState([...todos]);
```

Comment:
Could be simplified like this:
`  const [todoList, setTodoList] = useState(todos);`
The 'todos' you are importing are already an array of objects ready to be used. Since you are not modifying it initially, there is no need to use a spread operator.
Also name your variables using cammelCase for better readability: todoList instead of todolist